CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v1.0.0
+
+**This version contains breaking changes such as major version jumps in dependencies**
```

Comment:
Be more specific here, note the Ruby version change (-> 2.4).

Nitpicking: the TI-450 PR could have included a change to the gemspec:
> `s.required_ruby_version = '>= 2.4'`

Reply:
Updated the commit to include more detail about which breaking changes are included.